### PR TITLE
Add deployment scripts for HandlePackageEdits and ArchivePackages

### PR DESCRIPTION
**Pull request by scottbommarito**
This is the first step of deploying them on Octopus instead of manually!

also made some quick code style changes

### CODE REVIEW

**Comment by scottbommarito** [comment]
@skofman1 @ryuyu @shishirx34

File: src/ArchivePackages/Scripts/ArchivePackages.cmd [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@echo OFF
+
```
Comment (skofman1) [inline src/ArchivePackages/Scripts/ArchivePackages.cmd]: make sure it's the same as the vsts version

File: src/HandlePackageEdits/Scripts/HandlePackageEdits.cmd [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+cd bin
+
+:Top
+	echo "Starting job - #{Jobs.handlepackageedits.Title}"
+
+	title #{Jobs.handlepackageedits.Title}
+
+    start /w handlepackageedits.exe -VaultName "#{Deployment.Azure.KeyVault.VaultName}" -ClientId "#{Deployment.Azure.KeyVault.ClientId}" -CertificateThumbprint "#{Deployment.Azure.KeyVault.CertificateThumbprint}" -LogsAzureStorageConnectionString "#{Jobs.handlepackageedits.Storage.Primary}" -PackageDatabase "#{Jobs.handlepackageedits.PackageDatabase}" -SourceStorage "#{Jobs.handlepackageedits.SourceStorage}" -BackupStorage "#{Jobs.handlepackageedits.BackupStorage}" -Sleep #{Jobs.handlepackageedits.Sleep}
+
+	echo "Finished #{Jobs.handlepackageedits.Title}"
```
Comment (skofman1) [inline src/HandlePackageEdits/Scripts/HandlePackageEdits.cmd]: same

File: src/HandlePackageEdits/Scripts/PreDeploy.ps1 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+. .\Functions.ps1
+
+$jobsToInstall = $OctopusParameters["Jobs.ServiceNames"].Split("{,}")
+
+Write-Host Removing services...
+
+$jobsToInstall.Split("{;}") | %{
+	Uninstall-NuGetService -ServiceName $_
```
Comment (skofman1) [inline src/HandlePackageEdits/Scripts/PreDeploy.ps1]: LGTM

File: src/ArchivePackages/Scripts/ArchivePackages.cmd (hunk 1 above)
Comment (scottbommarito) [inline src/ArchivePackages/Scripts/ArchivePackages.cmd]: is same

File: src/HandlePackageEdits/Scripts/HandlePackageEdits.cmd (hunk 2 above)
Comment (scottbommarito) [inline src/HandlePackageEdits/Scripts/HandlePackageEdits.cmd]: is same

**Pull request merged**